### PR TITLE
Update de.Json [Patch-1] + Update word-filter.json [Patch-1] (DragonSinns)

### PR DESCRIPTION
**Pull request by Xx-DragonSinns-xX**

### CODE REVIEW

**Comment by Xx-DragonSinns-xX** [comment]
ups

**Comment by Xx-DragonSinns-xX** [comment]
howly mowly why i'm so bad at this 😢😂

File: contribution/lang/de.json [hunk 1]
```diff
@@ -3268,7 +3268,7 @@
  	 	 	 	"eng": "calibration"
  	 	 	},
  	 	 	"enemyGangsAndMark": {
- 	 	 	 	"trans": "Feinde Gangs und Ausrüstungsmerkmale",
+ 	 	 	 	"trans": "Gangs und Ausrüstungsmerkmale",
```
Comment (Pr0dam) [inline contribution/lang/de.json:3271]: This should read as Enemy gangs ( as its talking about the npc gang's Voodoo etc )

**Review by Pr0dam — CHANGES_REQUESTED** [review]
Just the one issue i can see, I left a comment on item 3271 so you should be able to see

**Comment by kt0t0Sudd3n** [comment]
I'll add those myself if you don't mind

**Comment by Xx-DragonSinns-xX** [comment]
@kt0t0Sudd3n 

No problem thanks :)

**Pull request merged**